Automattic/wp-calypso
# Email stats: update summary card styling

## Proposed Changes

<img width="1269" alt="CleanShot 2023-02-09 at 09 41 51@2x" src="https://user-images.githubusercontent.com/528287/217760747-c6c510f3-8f88-49a3-a394-a99d9216a1d6.png">

- Removes the bar from the Emails card in stats
- Changes the order of the Opens & Click columns
- Changes primary column's title to `Latest emails`

## Testing Instructions

- Apply this PR
Visit `http://calypso.localhost:3000/stats/day/<yoursite>?flags=newsletter/stats`
- The combined card should be in the bottom.

(This patch should also be applied to your sandbox: D100665-code )

## Pre-merge Checklist
